Userland/Libraries/LibWeb/CodeGenerators/WrapperGenerator.cpp

```diff
@@ -1311,7 +1317,7 @@ namespace Web::Bindings {
 void @prototype_class@::initialize(JS::GlobalObject& global_object)
 {
     [[maybe_unused]] auto& vm = this->vm();
-    [[maybe_unused]] u8 default_attributes = JS::Attribute::Enumerable | JS::Attribute::Configurable;
+    [[maybe_unused]] u8 default_attributes = JS::Attribute::Enumerable | JS::Attribute::Configurable | JS::Attribute::Writable;
```

Comment:
This might seem dangerous but you already cannot overwrite readonly attributes since they have no setter and from my testing I could overwrite everything in the big browsers

Reply:
Looking at the [spec](https://heycam.github.io/webidl/#es-interfaces) is much more useful than looking at other implementations :)

TL;DR: No, this is not correct in all cases, but neither was the previous code. Whatever, we can tighten this later.